src/ds5/ds5-private.h

```diff
@@ -132,8 +132,8 @@ namespace librealsense
             { RS430_PID,            "Intel RealSense D430"},
             { RS430_MM_PID,         "Intel RealSense D430 with Tracking Module"},
             { RS_USB2_PID,          "Intel RealSense USB2" },
-            { RS_RECOVERY_PID,      "Intel RealSense D4xx Recovery"},
-            { RS_USB2_RECOVERY_PID, "Intel RealSense USB2 D4xx Recovery"},
+            { RS_RECOVERY_PID,      "Intel RealSense D4XX Recovery"},
+            { RS_USB2_RECOVERY_PID, "Intel RealSense D4XX USB2 Recovery"},
```

Comment:
This is an API change, and affects our tests.
Need to discuss on  https://rsjira.intel.com/browse/DSO-17528